src/test/sighash_tests.cpp

```diff
@@ -234,7 +253,16 @@ BOOST_AUTO_TEST_CASE(sighash_from_data)
           stream >> tx;
 
           CValidationState state;
-          if (tx.nVersion < MIN_TX_VERSION) {
+          if (tx.fOverwintered) {
+              if (tx.nVersion == 3 && tx.nExpiryHeight > TX_EXPIRY_HEIGHT_THRESHOLD) {
```

Comment:
s/>/>=/

Reply:
Also add a test to make sure that the ``== TX_EXPIRY_HEIGHT_THRESHOLD`` case is exercised. (It isn't currently or this PR wouldn't have merged!)

Reply:
Filed #2959 to make sure this doesn't get lost.